lib/tiled/map.rb

```diff
@@ -43,6 +43,8 @@ def load
 
       # This is done last so that it can parse the object properties
       properties.from_xml_hash(custom_properties) if custom_properties
+
+      attributes.add(pixelwidth: width * tilewidth, pixelheight: height * tileheight)
```

Comment:
I think it is better to add this as methods, rather than attributes. I would prefer to leave attributes to TMX attributes only.